webpack/base.config.js

```diff
@@ -22,9 +22,12 @@ const baseConfig = {
   module: {
     loaders: [
       { test: /\.js/, loader: jsLoader, exclude: /node_modules/ },
+      { test: /\.json$/, loader: 'json-loader' },
```

Comment:
why is this part of this PR?

Reply:
Cherrio via enzyme loads json files apparently